doc/sphinx/language/gallina-specification-language.rst

```diff
@@ -1275,14 +1231,21 @@ constructions.
 
 .. _Fixpoint:
 
-.. cmd:: Fixpoint @ident @binders {? {struct @ident} } {? : @type } := @term
+.. cmd:: Fixpoint @fix_definition {* with @fix_definition }
+
+   .. insertprodn fix_definition decl_notation
+
+   .. prodn::
+      fix_definition ::= @ident_decl {* @binder } {? @fixannot } {? : @term } {? := @term } {? @decl_notations }
+      decl_notations ::= where @decl_notation {* and @decl_notation }
+      decl_notation ::= @string := @term1_extended {? : @ident }
 
    This command allows defining functions by pattern matching over inductive
    objects using a fixed point construction. The meaning of this declaration is
-   to define :token:`ident` a recursive function with arguments specified by
-   the :token:`binders` such that :token:`ident` applied to arguments
-   corresponding to these :token:`binders` has type :token:`type`, and is
-   equivalent to the expression :token:`term`. The type of :token:`ident` is
+   to define :n:`@ident` a recursive function with arguments specified by
+   the :n:`@binders` such that :n:`@ident` applied to arguments
```

Comment:
Everywhere there is a reference to `binders` it should become ```:n:`@binder`\s``` since the latter does not exist anymore.

Reply:
It would helpful if Sphinx generated an error message for this type of missing reference

Reply:
It does though, but you have disabled it by adding `binders` to the `nitpick_ignore` list in `conf.py`.